index.html

```diff
@@ -0,0 +1,166 @@
+    <link rel="icon" href="./Images/logo-no-background.svg" type="image/svg" />
+  </head>
+  <body>
+    <aside class="aside-section">
+      <div class="top-elements">
+          <i class="fa-brands fa-facebook-f aside-icon"></i>
+          <i class="fa-brands fa-twitter aside-icon"></i>
+          <a href="#">English</a>
+          <a href="#">Page</a>
+          <a href="#">Logout</a>
+      </div>
+    </aside>
+    <section class="nav-bar">
+      <div id="ham-menu">
+        <img
+          onclick="clickX()"
+          id="x"
+          src="https://img.icons8.com/sf-regular/48/FFFFFF/multiply.png"
+        />
+        <ul class="ham">
+          <li>
+            <a href="#" class="ham-items" onclick="clickX()"
+              >Home</a
+            >
+          </li>
+          <li>
+            <a href="about.html" class="ham-items" onclick="clickX()">About</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Program</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Join</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Sponsor</a>
+          </li>
+        </ul>
+      </div>
+      <header class="head flex-row">
+        <nav>
+          <img
+            onclick="clickMenu()"
+            id="menu"
+            src="./Images/Icon - Menu.png"
+            alt="Menu Logo"
+            class="menu-logo"
+          />
+          <a href="">
+            <img class="logo" src="Images/logo-black.png" alt="logo" />
+          </a>
+          <ul class="menu_items flex-row">
+            <li class="nav_items"><a href="#">Home</a></li>
+            <li class="nav_items"><a href="about.html">About</a></li>
+            <li class="nav_items"><a href="">Program</a></li>
+            <li class="nav_items"><a href="">Join</a></li>
+            <li class="nav_items"><a href="">Sponsor</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div class="main-headline">
+        <h2 class="welcome">Welcome Humans</h2>
+        <h3 class="welcome-2">World is moving with</h3>
+        <h1 class="Meco">MECO</h1>
+        <p>
+          Bringing the 
+          change together 
+          with A.I. What you show They do learn 
+          The tech world is constantly evolving,with technologies
+          emerging regularly. <br>You are a Part of them <br>tooo!
+        </p>
+        <h3 class="bottom">
+          We are glad to share you that A joyful celebration believing in the
+          value of Machine Learning and A.I., creating a positive change with
+          people from over 80 countries is taking place in October, in Korea.
+        </h3>
+        <h2 class="date">2015.10.15(THU) ~ 16(FRI)</h2>
+        <h3 class="venue">@ Dhaka, Bangladesh
+        </h5>
+      </div>
+    </section>
+    <section class="program">
+        <h2 class="section-heading">Main Program</h2>
```

Comment:
- [ ] I noticed that the `Main Program` section of your design is bleeding into the `Headline Section`, causing it to look significantly different from the original design. It also looks like you don't have content in the `Main Program` section. This is affecting the overall look and feel of your design, which is not ideal. I will suggest that you ensure that you add all the necessary content needed for that section and ensure that the design follows the given design guidelines as the previous reviewer mentioned :+1: 
- [ ] It should have a section header, a divider, five individual articles, each with an image, title, and description, and a line of text at the bottom as seen in the original design :wink: 
- [ ] There is a similar issue in the mobile view as well so please ensure that you fix these in both mobile and desktop and ensure that they both follow their respective design guidelines :wink: 
- Your Design Desktop
![image](https://user-images.githubusercontent.com/102699958/233694473-7f0b3c5c-f201-4d4e-b3db-b3a40a2f9edb.png)

- The Original Design Desktop
![image](https://user-images.githubusercontent.com/102699958/233694745-0486463c-9a93-4acc-a340-01642fd1f8a2.png)

| Your Mobile Design | The Original Mobile Design |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/102699958/233695279-28fffa8d-b4de-4cae-88d9-718aff3b9f56.png) | ![image](https://user-images.githubusercontent.com/102699958/233695532-4daa4ea4-57ce-4002-85fd-3da6d230cceb.png) |